### PR TITLE
Add bumpversion config

### LINKED ISSUE #62: Use bumpversion
In the comments at https://blog.ionelmc.ro/2014/05/25/python-packaging/ Ionel Cristian Mărieș mentions using [bumpversion](https://pypi.python.org/pypi/bumpversion) to manage the versioning of libraries and his cookiecutter templates include the configuration of that tool.

There's also a nice write-up of it by Atlassian: https://developer.atlassian.com/blog/2016/02/bumpversion-is-automation-for-semantic-versioning/

I suggest we replace `version.py` and use this tool instead.

### PR DESCRIPTION
**Pull request by meatballs**
Fixes #62

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,13 +1,8 @@
...
 
-# Read in the version number
-__version__ = None  # This makes linters happy
-with open('conference_scheduler/version.py', 'r') as f:
-    exec(f.read())
-
 setup(
     name='conference-scheduler',
-    version=__version__,
+    version='1.1.0',
```
Comment (drvinceknight) [inline setup.py]: The  idea is that `bumpversion` will rewrite this line for us? (I'm happy either way just making sure it's supposed to be in both places).

Comment (meatballs) [inline setup.py]: Yes - and I've tested locally that it does

Comment (meatballs) [inline setup.py]: It knows to do that because of L14 in `setup.cfg`

Comment (drvinceknight) [inline setup.py]: :+1: Neat. I'll merge, I think coverage is having a bad day.

**Pull request merged**